src/style.css

```
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: block;
+}
+
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(./star-active.svg);
+}
+
+.stars .stars__star:not(:last-child) {
+  margin-right: 4px;
+}

```

```suggestion
.stars__star:not(:last-child) {
  margin-right: 4px;
}

```
No need to increase the specificity